src/components/list/list.js

```diff
@@ -86,7 +90,7 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
         }
         if (hasProxiedElement) {
           wrapIn('div');
-        } else if (!tEl[0].querySelector('md-button')) {
+        } else if (!tEl[0].querySelector('md-button:not(.md-secondary):not(.md-exclude)')) {
```

Comment:
It's documented on lines 66-69 above which appear as a note in the directive API docs. Does it need to go somewhere else too?

> and you can include a class of `class="md-exclude"` if you need to use a non-secondary button